src/test/daml/Daml/Finance/Instrument/Option/Test/Util.daml

```diff
@@ -60,11 +61,12 @@ originateDividendOption : Party -> Party -> Text -> Text -> [(Text, Parties)] ->
   -> Script InstrumentKey
 originateDividendOption depository issuer label description observers lastEventTimestamp
   expiryDate cashQuantity sharesQuantity fxQuantity = do
+    let prevElections = []
   -- CREATE_DIVIDEND_OPTION_INSTRUMENT_BEGIN
     cid <- toInterfaceContractId <$> submitMulti [depository, issuer] [] do
       createCmd DividendOption.Instrument with
         depository; issuer; id = Id label; version = "0"; description
-        observers = M.fromList observers; lastEventTimestamp
+        observers = M.fromList observers; lastEventTimestamp; prevElections
```

Comment:
can simplify with `prevElections = []`

Reply:
I fixed this as part of this PR: https://github.com/digital-asset/daml-finance/pull/779